OutOfSchool/OutOfSchool.WebApi/Services/Database/WorkshopService.cs

```diff
@@ -245,6 +251,35 @@ public async Task<WorkshopDTO> Update(WorkshopDTO dto)
             return mapper.Map<WorkshopDTO>(currentWorkshop);
         }
 
+        /// <inheritdoc/>
+        public async Task<WorkshopDTO> UpdateStatus(Guid id, WorkshopStatus status)
+        {
+            logger.LogInformation($"Updating Workshop status with Id = {id} started.");
+
+            var currentWorkshop = await workshopRepository.GetWithNavigations(id).ConfigureAwait(false);
```

Comment:
We do not need all of the workshop related information here. This call can be changed.
```suggestion
            var currentWorkshop = await workshopRepository.GetById(id).ConfigureAwait(false);
```